tests/test_particular_profiles.py

```diff
@@ -271,3 +272,32 @@ def test_simple_profile_holiday_is_sunday(self):
         assert np.isclose(self.ilp_holiday.dataframe["ind"].sum(), 98.9)
         # check total demand
         assert np.isclose(df.sum(), 1.0)
+
+
+class TestHeatBuilding:
+    @classmethod
+    def setup_class(cls):
```

Comment:
Working with a test class does only make sense if there are more than one test functions using that class.